themes/tiddlywiki/vanilla/base.tid

```diff
@@ -92,7 +91,7 @@ pre {
 	white-space: pre-wrap;
 	background-color: <<colour pre-background>>;
 	border: 1px solid <<colour pre-border>>;
-	padding: 0 3px 2px;
+	padding: 0.5em;
```

Comment:
As the heading says - to match the padding of .hljs elements. I did not think the change was a drastic one. It makes pre elements and highlighted code pieces have the same padding.
@Jermolene, Feel free to close this if not willing to merge. There are enough "dangling" pull requests already :).